docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -2503,16 +2503,24 @@ end
 | Yes
 | No
 | 1.0
-| -
+| 2.4
 |===
 
-Checks for multiple top-level example groups.
+Checks for multiple top-level example groups or nested example groups.
+
+This cop can be configured using the `EnforcedStyle` option.
+
+Using the `EnforcedStyle: nesting`, multiple descriptions for the
+same class or module should either be nested or separated into different
+test files.
 
-Multiple descriptions for the same class or module should either
-be nested or separated into different test files.
+Using the `EnforcedStyle: splitting`, nested describe for the same class
+or module must be either nested or split into separate test files.
```

Comment:
Are you certain this is what the cop suggests in the `splitting` mode?

Reply:
The `nested` word seems unnecessary. I corrected the sentence.